### PR TITLE
ensemblgenomes: add additional URL

### PR DESCRIPTION
**Pull request by bernt-matthias**
since the ensembl and ensemblgenomes ftp seem to be analogous I just added it like so

### CODE REVIEW

File: ensembldl.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ USAGE
 
 DESCRIPTION
 
-  download ENSEMBL data set
+  download ENSEMBL prefix dataset
```
Comment (wookietreiber) [inline ensembldl.sh]: This is the description of the tool, not the command line usage. The usage is directly above and contains the prefix. Please remove this change.

Comment (bernt-matthias) [reply ensembldl.sh]: done

**Comment by wookietreiber** [comment]
Waiting for Travis to finish. Otherwise, LGTM.

**Comment by bernt-matthias** [comment]
time limit? I tried to choose small data sets. Hope I did not make a mistake. Do you want to restart the test?

**Comment by wookietreiber** [comment]
Restarted.

**Comment by bernt-matthias** [comment]
Hmm. Selbes problem...

**Comment by wookietreiber** [comment]
I restarted and it crashed again. Looks like Travis doesn't wanna talk to ftp://ftp.ncbi.nlm.nih.gov. I'll try to open an issue with Travis.

**Comment by wookietreiber** [comment]
https://blog.travis-ci.com/2018-07-23-the-tale-of-ftp-at-travis-ci

**Comment by bernt-matthias** [comment]
OK. Lets try if sftp works...

**Comment by bernt-matthias** [comment]
Odd thing is that the blog post is from 2018 where the tests seemed to work.

**Comment by wookietreiber** [comment]
NCBI does not provide SFTP.

I think, we need to drop Travis for the moment. Neither of their solutions works well for us, i.e. not all sites support SFTP and/or HTTPS and VPN is not a solution I'm comfortable with. 

I know that NCBI is also accessible via HTTPS, but I'm not willing ATM to offer a choice between protocols in the download tools. This is too much effort.

**Comment by wookietreiber** [comment]
I have disabled Travis for now. Please drop the SFTP commit and force push. Then, I guess the Travis check will disappear.

**Comment by wookietreiber** [comment]
Nevermind, I have just merged myself and fixed a few things.

**Pull request closed without merging**